### ISSUE TITLE
aria-expanded value not getting updated on openSubmenu and closeSubmenu callbacks.

### ISSUE THREAD

**Issue by ajmaurya99** [10up/component-library#106]
**Describe the bug**
The Aria Expanded value is not getting updated when clicked on another menu items.

**To Reproduce**
1. Open the Navigation dropdown menu item.
2. Click on the sibling menu item.
3. The aria-expanded value remains unchanged and is not updated based on the openSubmenu and closeSubmenu callbacks.

**Expected behavior**
The aria-expanded value of the anchor element should be updated based on the openSubmenu and closeSubmenu callbacks

**Screenshots**
[If applicable, add screenshots to help explain your problem.](https://share.getcloudapp.com/rRu16K7Q)

**Desktop (please complete the following information):**

**Mobile (please complete the following information):**

**Additional context**
Please list more relevant information here.
